bench/basho_bench_driver_poolgirl.config

```diff
@@ -1,21 +0,0 @@
-{driver, basho_bench_driver_poolgirl}.
```

Comment:
The benchmark code is broken, unrequired and pulls a lot of crap just for a simple test.